diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0485b041a1..3e8ba7a434 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -89,9 +89,6 @@ if(NOT CMake_TEST_EXTERNAL_CMAKE)
   # check for available C++ features
   include(${CMake_SOURCE_DIR}/Source/Checks/cm_cxx_features.cmake)
 
-  if(NOT CMake_HAVE_CXX_UNIQUE_PTR)
-    message(FATAL_ERROR "The C++ compiler does not support C++11 (e.g. std::unique_ptr).")
-  endif()
 endif()
 
 # Inform STL library header wrappers whether to use system versions.
